core/interface/static/css/projectalice.css

```diff
@@ -1,714 +1,246 @@
-	src: url('../webfonts/marske-webfont.woff2') format('woff2'),
-	url('../webfonts/marske-webfont.woff') format('woff');
+	font-family: 'Roboto-Slab';
+	src: url('../webfonts/roboto-slab-v10-latin/roboto-slab-v10-latin-regular.eot'); /* IE9 Compat Modes */
+	src: url('../webfonts/roboto-slab-v10-latin/roboto-slab-v10-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+	url('../webfonts/roboto-slab-v10-latin/roboto-slab-v10-latin-regular.woff') format('woff'), /* Modern Browsers */
+	url('../webfonts/roboto-slab-v10-latin/roboto-slab-v10-latin-regular.ttf')  format('truetype'), /* Safari, Android, iOS */
+	url('../webfonts/roboto-slab-v10-latin/roboto-slab-v10-latin-regular.svg#svgFontName') format('svg'); /* Legacy iOS */
 	font-weight: normal;
 	font-style: normal;
-}
-
+  }
 @font-face {
-	font-family: 'marskeregular';
-	src: url('../webfonts/marske-webfont.woff2') format('woff2'),
-	url('../webfonts/marske-webfont.woff') format('woff');
+	font-family: 'Roboto';
+	src: url('../webfonts/roboto-v20-latin/roboto-v20-latin-regular.eot'); /* IE9 Compat Modes */
+	src: url('../webfonts/roboto-v20-latin/roboto-v20-latin-regular.eot#iefix') format('embedded-opentype'), /* IE6-IE8 */
+	url('../webfonts/roboto-v20-latin/roboto-v20-latin-regular.woff') format('woff'), /* Modern Browsers */
+	url('../webfonts/roboto-v20-latin/roboto-v20-latin-regular.ttf')  format('truetype'), /* Safari, Android, iOS */
+	url('../webfonts/roboto-v20-latin/roboto-v20-latin-regular.svg#svgFontName') format('svg'); /* Legacy iOS */
 	font-weight: normal;
 	font-style: normal;
 }
-
 @font-face {
-	font-family: 'neogreyregular';
-	src: url('../webfonts/neogreyregular-webfont.woff2') format('woff2'),
-	url('../webfonts/neogreyregular-webfont.woff') format('woff');
+	font-family: 'Lato';
+	src: url('../webfonts/lato-v16-latin/lato-v16-latin-regular.eot'); /* IE9 Compat Modes */
+	src: url('../webfonts/lato-v16-latin/lato-v16-latin-regular.eot#iefix') format('embedded-opentype'), /* IE6-IE8 */
+	url('../webfonts/lato-v16-latin/lato-v16-latin-regular.woff') format('woff'), /* Modern Browsers */
+	url('../webfonts/lato-v16-latin/lato-v16-latin-regular.ttf')  format('truetype'), /* Safari, Android, iOS */
+	url('../webfonts/lato-v16-latin/lato-v16-latin-regular.svg#svgFontName') format('svg'); /* Legacy iOS */
 	font-weight: normal;
 	font-style: normal;
 }
-
-html,
-body {
-	background-color: black;
-	width: 100%;
-	margin: 0 auto auto;
-	color: #d1d1d1;
+@font-face {
+	font-family: 'Roboto-Light';
+	src: url('../webfonts/Roboto-Light.ttf') format('truetype');
+	font-weight: normal;
+	font-style: normal;
 }
 
-header {
-	font-family: 'marskeregular', Arial, sans-serif;
-	height: 50px;
-	background-color: #1e1e1e;
-	margin: 2px;
-	color: #d1d1d1;
-	display: flex;
+:root {
+    --primary-color: #eeeeee;
+    --secondary-color: #101014;
+    --log-bg-color: #101014;
+	--bg-color: #fbfbff;
+	--ui-bg-color: #1a1a29;
+	--transit-color: #6f788f;
+	--inv-bg-color:#343647;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected RBRACE at line 45, col 2.](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4618587)

Reply:
This issue is no issue... its just a use of variables.